bencher/test/src/lib.rs

```diff
@@ -77,12 +77,12 @@ pub mod pallet {
 
 		#[orml_weight_meter::weight(0)]
 		pub(crate) fn remove_all_bar() {
-			Bar::<T>::remove_all(None);
+			_ = Bar::<T>::clear(10, None);
```

Comment:
should it be `_ = Bar::<T>::clear(u32::max_value(), None)`?

Reply:
no, it's just test

Reply:
I made it optional because now it's required to pass a number so it's optional if you want to count that as writes